### ISSUE TITLE
npm for non-webpack casts

### ISSUE THREAD

**Issue by perborgen** [scrimba/community#81]
If you create a new cast and choose e.g. JavaScript as template, the npm stuff doesn't work (isn't using index.pack.js). We need to communicate clearly to the user if they can use npm in the cast.

**Comment by somebee** [comment]
I agree. We should use .pack.js extension by default in all templates. One solution would be to log out a warning "to use require/import you need to use (filename).pack.js"  if you try loading a regular .js-file (that uses require/import).

**Comment by judofyr** [comment]
We can define `require` in plain JS files to `console.log` that error.

**Comment by perborgen** [comment]
We should however also have some templates that are *exactly* as how you'd work locally. (So that the .zip export will result in the same behavior if you try it out locally). This important for some types of tutorials for beginners.

**Comment by judofyr** [comment]
I've pushed (but not deployed) that causes `require()` to error with a proper message:

<img width="622" alt="screen shot 2017-09-08 at 16 54 52" src="https://user-images.githubusercontent.com/499/30217457-7bde600c-94b6-11e7-8832-f7364bd78cc7.png">

This doesn't work when you use native `import`-syntax though, since the file fails to even parse in the browser.

**Comment by somebee** [comment]
Maybe we should consider defaulting to using packed version, and then rather expose the non-packed version as (myfile).js.raw? I guess in 95% of the cases, people would like to use the packed version that takes care of babelifying etc.
